### PR TITLE
the menu Icons have been changed to vector-based images.

### PR DESCRIPTION
**Pull request by rojinshahbazian**

### CODE REVIEW

File: app/src/main/AndroidManifest.xml [hunk 1]
```diff
@@ -31,9 +31,9 @@
         </activity>
 
 
-        <meta-data
+       <!-- <meta-data
             android:name="com.google.android.geo.API_KEY"
-            android:value="@string/api_key"/>
+            android:value="@string/api_key"/>-->
```
Comment (Andrew-Delgado) [inline app/src/main/AndroidManifest.xml]: Uncomment this code

File: app/src/main/res/menu/activity_main_drawer.xml [hunk 2]
```diff
@@ -6,44 +6,44 @@
...
         <item
             android:id="@+id/nav_contacts"
-            android:icon="@drawable/ic_menu_gallery"
+            android:icon="@drawable/ic_contacts"
             android:title="@string/menu_contacts" />
 
         <item
             android:id="@+id/nav_add_contact"
-            android:icon="@drawable/ic_menu_manage"
+            android:icon="@drawable/ic_add_contacts"
```
Comment (Andrew-Delgado) [inline app/src/main/res/menu/activity_main_drawer.xml]: The compiler can not find this drawable. I do not see it in any folder.

**Review by Andrew-Delgado — CHANGES_REQUESTED** [review]
I left some comments on the changes that need to be done. Overall the icons look good.

**Comment by rojinshahbazian** [comment]
made changes since last pull request

**Pull request merged**